src/utils/config.ts

```diff
@@ -1 +1 @@
-export const aspectRatio = 7 / 1;
+export const aspectRatio = 7 / 2;
```

Comment:
^ need to see if this is better size for all screen sizes

Reply:
looks better